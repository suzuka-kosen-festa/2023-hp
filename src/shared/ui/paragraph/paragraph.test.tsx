import { composeStory } from "@storybook/react";
import { render } from "@testing-library/react";
import Meta, { Default } from "./paragraph.stories";

describe("[shared/ui/paragraph] paragraph.tsx", () => {
  it("should render the Paragraph component", () => {
    const Composed = composeStory(Default, Meta);
    const { container } = render(<Composed />);
    expect(container).toMatchSnapshot();
  });
});
