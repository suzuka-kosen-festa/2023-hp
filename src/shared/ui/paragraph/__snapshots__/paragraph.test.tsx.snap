// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[shared/ui/paragraph] paragraph.tsx > should render the Paragraph component 1`] = `
<div>
  <p
    class="styles_p__62gm1o0"
  >
    test test test
  </p>
</div>
`;
